src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// write code here
+const sortList = (list) => {
+  [...list].map((li) => {
+    const salary = li
+      .getAttribute('data-salary')
+      .slice(1, -1)
+      .replace(',', '.');
```

Comment:
The salary attribute is being sliced and replaced, but it is not being converted to a number. This might cause issues when sorting, as string comparison will be used instead of numerical comparison. Consider using `parseFloat` to convert the salary to a number.